libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -57,28 +59,7 @@ class PredictiveCacheController @JvmOverloads constructor(
     private val navigation: MapboxNavigation,
     private val predictiveCacheControllerErrorHandler: PredictiveCacheControllerErrorHandler? = null
 ) {
-    private var map: MapboxMap? = null
-
-    private val onStyleLoadedListener = object : OnStyleLoadedListener {
-        override fun onStyleLoaded() {
-            map?.let { map ->
-                val tileStore = map.getResourceOptions().tileStore
-                if (tileStore == null) {
-                    handleError("TileStore instance not configured for the Map.")
-                    return
-                }
-                val currentMapSources = mutableListOf<String>()
-                traverseMapSources(map) { tileVariant ->
-                    currentMapSources.add(tileVariant)
-                }
-                updateMapsControllers(
-                    currentMapSources,
-                    PredictiveCache.currentMapsPredictiveCacheControllers(),
-                    tileStore
-                )
-            }
-        }
-    }
+    private var mapListeners = mutableMapOf<MapboxMap, OnStyleLoadedListener>()
```

Comment:
Similar question / comment to the one above. Do we really need a `MapboxMap` dictionary? 🤔

Reply:
answered above.